### ISSUE TITLE
Does not work with any district owned schoology

### ISSUE THREAD

**Issue by shedaoo878** [jsadowyj0/schoology-ext#1]
Our schoology for school has an fuhsd domain so the extension does not work

**Comment by jsadowyj0** [comment]
Yeah, I've had to deal with this issue before. The thing is its either I can let the extension have access to every website or only Schoology related domains. I went with the second option as I didn't want errors popping up in everyone's console all the time. I tried to fix this with wildcards for url matches but to no avail. Well, anyway, what's your schools domain? I can just add it to the manifest so it matches every time.

Also, for anyone looking at this issue feel free to make a pull request with your school's domain, as I can't really predict all the possible custom domains.

**Comment by shedaoo878** [comment]
The domain is fuhsd

**Comment by jsadowyj0** [comment]
Does the url look like https://schoology.fuhsd.com? I need an exact url.

**Comment by shedaoo878** [comment]
fuhsd.schoology.com

**Comment by jsadowyj0** [comment]
Well, bad news, I just looked over the codebase again and even if I add your domain it still won't work because of the way Schoology just redid their website. So for now I just need to deprecate this whole extension until I can come out with a fix, which could be a while from now depending on how school goes.

I'm sorry for the inconvenience, but I'll leave this code on this repo if you ever want to try and make something like this yourself.

**Comment by FlowerGirl5354** [comment]
Hi. I can not use this extension because of my school Schoology domain. Is there any way that you can add this domain to the extension? Thank you. https://cfisd.schoology.com/home#/?_k=9vgc97

**Comment by jsadowyj0** [comment]
Yep, I'll add all of these right now. Just to make sure though, the extension is working when you use the app.schoology.com domain correct?

**Comment by FlowerGirl5354** [comment]
I haven't tested it because I do not have a app.schoology.com login.

**Comment by jsadowyj0** [comment]
Ok, no problem I'll just add it anyway. Now I'm waiting for the other person to respond with their domain then I'll push out and update. Thanks for letting me know!

**Comment by shedaoo878** [comment]
fuhsd.schoology.com

**Comment by jsadowyj0** [comment]
I should have already added that one, does it not work still?

**Comment by FlowerGirl5354** [comment]
That domain is not mine. Right now it just says to input assignment IDs

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by jsadowyj0** [comment]
I know it isn't yours, I added yours too. I just wanted to wait for the other guy to respond before I actually pushed out the changes to the chrome store. But it seems like he is not responding so I will add your schools domain and release an update.

**Comment by Byrdman32** [comment]
School Domain littlerocksd.schoology.com
